studies/india_districts/requirements.txt

```diff
@@ -0,0 +1,56 @@
+-e git+https://github.com/mansueto-institute/adaptive-lockdown@574e31d27273f90b5c9a01bf11f9292f4cc24b25#egg=adaptive
```

Comment:
move the output of pip freeze to the repo root

Reply:
sorry I thought I had done this